Automattic/wp-calypso
# Plans: Add new feature flag to gate changes to Jetpack products

#### Changes proposed in this Pull Request

* Create a new feature flag called `jetpack/only-realtime-products`, setting its default value to `false` in all environments.
* Re-alphabetize Jetpack-related feature flags, so they can be searched more easily.

#### Testing instructions

* In any Calypso environment, visit the Plans/Pricing page.
* Verify that the page looks the same as it currently does in production.
* Add `?flags=jetpack/only-realtime-products` to your browser window's URL and hit Enter to re-load the page.
* Verify that you see a header saying the feature flag is now enabled.

#### Reference screenshots (disabled / enabled)

<img height="200" alt="image" src="https://user-images.githubusercontent.com/670067/123813018-2c14a800-d8ba-11eb-97cb-89be1060e6f7.png"> <img height="200" alt="image" src="https://user-images.githubusercontent.com/670067/123813639-b1985800-d8ba-11eb-8d4a-f4f6c8413cdc.png">
